extensions/wikia/AdEngine/js/provider/evolve2.js

```diff
@@ -6,8 +6,7 @@ define('ext.wikia.adEngine.provider.evolve2', [
 	'ext.wikia.adEngine.utils.adLogicZoneParams',
 	'ext.wikia.adEngine.utils.eventDispatcher',
 	'wikia.log',
-	require.optional('ext.wikia.adEngine.lookup.openx.openXBidderHelper')
-], function (adContext, gptHelper, slotTweaker, zoneParams, eventDispatcher, log, openXHelper) {
+], function (adContext, gptHelper, slotTweaker, zoneParams, eventDispatcher, log) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 6 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)